src/utility.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as opn from "opn";
+import * as vscode from "vscode";
+import { logger, Type } from "./logger";
+
+function openLearnMoreLink(message: string): void {

```

Comment:
Suggest generalizing this function by changing it to openLink and passing the url as a parameter.

Reply:
fixed.